### DISCUSSION TITLE
Why fillna function is adding new rows to dataset

### DISCUSSION THREAD

**Discussion by shahbaz-prog** [AammarTufail/python_ka_chilla2023#52] [Q&A]
<img width="824" alt="Question" src="https://user-images.githubusercontent.com/119630744/211698549-7d3b5f46-d08f-4e59-8740-5679cb8e0cda.png">
<img width="761" alt="Question_2" src="https://user-images.githubusercontent.com/119630744/211698554-5ff3a893-c56d-4ac4-861a-40f3508a455f.png">

**Comment by Masoodullah909** [comment]
Because you define in a function where a (Null Value / NA / Empty cell) exists
 (replace it / fill) it by zero.
